statistics/handle/cache/internal/inner.go

```diff
@@ -26,7 +26,7 @@ type StatsCacheInner interface {
 	// Get gets the cache.
 	Get(tid int64, moveFront bool) (*statistics.Table, bool)
 	// Put puts a cache.
-	Put(tid int64, tbl *statistics.Table, moveLRUFront bool)
+	Put(tid int64, tbl *statistics.Table, moveLRUFront bool) bool
```

Comment:
In our scenarios, it seems that we don't need to expose the `bool`.

Reply:
Actually, in the ```StatsCache```, the retry logic is already included, so the internal cache needs to expose its status for ```StatsCache``` to handle retries and logging.